lib/modules/manager/crossplane/schema.ts

```diff
@@ -1,7 +1,8 @@
 import { z } from 'zod';
+import { regEx } from '../../../util/regex';
 
 export const XPKGSchema = z.object({
-  apiVersion: z.string().regex(/^pkg\.crossplane\.io\//),
+  apiVersion: z.string().regex(regEx(/^pkg\.crossplane\.io\//)),
```

Comment:
isn't there a startsWith alternate? that would be faster than regex

Reply:
Maybe, I will take a look and add if there is one in another PR